client/my-sites/domains/domain-management/list/empty-domains-list-card.jsx

```diff
@@ -20,7 +26,7 @@ function EmptyDomainsListCard( { selectedSite, hasDomainCredit, isCompact, hasNo
 
 	const siteHasHundredYearPlan = selectedSite?.plan?.product_slug === PLAN_100_YEARS;
 
-	let title = translate( 'Get your domain' );
+	let title = translate( 'Get your free domain' );
```

Comment:
It would be nice to add a translation check here.
```
	let title = i18n.hasTranslation( 'Get your free domain' )
		? translate( 'Get your free domain' )
		: translate( 'Get your domain' );
```

Reply:
Agreed, that's a quick quality win.